index.js

```diff
@@ -44,4 +44,9 @@ function patternImporter (options) {
 
 }
 
-module.exports = patternImporter;
+module.exports = exports = patternImporter;
```

Comment:
I think I want to switch to this:
```
module.exports = {
  patternImporter: patternImporter,
  utils: utils
}
```

for clarity and to allow us to open up other functions to use via NPM